src/kit/Progress.tsx

```diff
@@ -60,7 +60,7 @@ const Progress: React.FC<Props> = ({
       <div className={classes.join(' ')}>
         <div
           className={css.bar}
-          style={{ height: `calc(${sizeMap[size]} + var(--theme-density) * 1px)` }}>
+          style={{ height: `calc(calc(${sizeMap[size]} * 1px) + var(--theme-density) * 1px)` }}>
```

Comment:
would this work instead?
```suggestion
          style={{ height: `calc((${sizeMap[size]} + var(--theme-density)) * 1px)` }}>
```

Reply:
Yea lets go with that!